xml/System.Security.Cryptography.X509Certificates/CertificateRequest.xml

```diff
@@ -388,9 +388,17 @@ The type of signing key represented by <paramref name="issuerCertificate" /> cou
 
  -or-
 
-<paramref name="issuerCertificate" /> has a different key algorithm than the requested certificate.</exception>
+<paramref name="issuerCertificate" /> has a different key algorithm than the requested certificate.
+
+-or-
+
+The <see cref="P:System.Security.Cryptography.X509Certificates.X509Certificate2.HasPrivateKey" /> value for <paramref name="issuerCertificate" /> is <see langword="false" />.</exception>
         <exception cref="T:System.InvalidOperationException">
-          <paramref name="issuerCertificate" /> is an RSA certificate and the current object was created using a constructor that doesn't accept a <paramref name="padding" /> parameter.</exception>
+          <paramref name="issuerCertificate" /> is an RSA certificate and the current object was created using a constructor that doesn't accept a <paramref name="padding" /> parameter.
+
+-or-
+
+<paramref name="issuerCertificate" /> is an RSA certificate and this object was created via a constructor that doesn't accept a <see cref="T:System.Security.Cryptography.RSASignaturePadding" /> value.</exception>
```

Comment:
This sentence and the one before it are the same intent with slightly different wording.  We should only have one of them.

Reply:
Thanks for clarifying. When I ported it, I wasn't entirely sure if they were talking about the same, I wanted you to confirm.
I'll remove it.